src/test/java/stepDefinitions/searchSteps.java

```diff
@@ -2,62 +2,55 @@
 
```

Comment:
why this class has a lot of modifications?

Reply:
Alternative solutions were tested to pass the browser as a parameter. Were discarded